src/autocomplete/typeahead.js

```diff
@@ -39,6 +39,8 @@ function Typeahead(o) {
 
   this.css = o.css = _.mixin({}, css, o.appendTo ? css.appendTo : {});
   this.cssClasses = o.cssClasses = _.mixin({}, css.defaultClasses, o.cssClasses || {});
+  this.cssClasses.prefix =
+    o.cssClasses.formattedPrefix = _.formatPrefix(this.cssClasses.prefix, this.cssClasses.noPrefix);
```

Comment:
If tests are fine then I am fine clearly, you may have a better understanding of this after spending some time on it than me.

Is there anything you can think could go wrong?